src/main/java/org/zalando/nakadi/controller/SubscriptionStreamController.java

```diff
@@ -203,20 +202,18 @@ private StreamingResponseBody stream(final String subscriptionId,
         final String flowId = FlowIdUtils.peek();
 
         return outputStream -> {
-            final Scope scope = TracingService.activateSpan(parentSubscriptionSpan, false);
-            FlowIdUtils.push(flowId);
-            scope.span().setOperationName("stream_events_request");
-            scope.span().setTag("subscription.id", subscriptionId);
-            scope.span().log(String.format("Stream Parameters: batchLimitEvents: {}," +
+            TracingService.activateSpan(parentSubscriptionSpan, false)
+                    .setOperationName("stream_events_request")
+                    .setTag("subscription.id", subscriptionId)
+                    .log(String.format("Stream Parameters: batchLimitEvents: {}," +
```

Comment:
it is better to extract String.format to a variable, and as far as i remember it uses `%s` vs `{}`

Reply:
https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html